studio/components/SalaryCsvInput/SalariesParseErrorsToastDescription.tsx

```diff
@@ -0,0 +1,36 @@
+import {
```

Comment:
I'd omit this.

It's a good idea, but web doesn't natively support toasts. There's a lot of added configuration needed in order to truly make toasts accessible on all platforms, regardless of what a11y tech the user is using. 

The sanity team has gotten a lot of feedback and requests for a better error handling inside the studio — and I'm hoping they're adding a fix to one of their releases soon.

meanwhile, it's a possible for users to check this little icon:
<img width="341" alt="Screenshot 2024-09-03 at 10 48 38" src="https://github.com/user-attachments/assets/b88ce427-70c4-494e-91f5-5b119027ce22">
to see all available error messages:
<img width="1030" alt="Screenshot 2024-09-03 at 10 49 27" src="https://github.com/user-attachments/assets/68c77751-f568-4a05-8d06-97ba02472a4b">